### PR TITLE
Add `wixstudio.com`

### PR DESCRIPTION
**Pull request by alonkochba**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Wix is a website building platform.

Registered developers can create apps and get their own subdomain (e.g. username.wixstudio.com) on which their sites are hosted.

Organization Website: https://www.wix.com/

Reason for PSL Inclusion
====

We'd like to properly isolate each subdomain to prevent global cookie setting on wixstudio.com and highlight that each subdomain hosts sites of a different user, unrelated to the others.

Number of users this request is being made to serve:

DNS Verification via dig
=======

> dig +short TXT _psl.wixstudio.com 
https://github.com/publicsuffix/list/pull/1971

Results of Syntax Checker (`make test`)
=========
```
============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================

```

### CODE REVIEW

**Comment by alonkochba** [comment]
@shahata

**Comment by yahesh** [comment]
@alonkochba Please order your block correctly. Entries are order by TLD first and then by the SLD (so-called [right-to-left sorting](https://github.com/publicsuffix/list/wiki/Format#right-to-left-sorting).

**Comment by yahesh** [comment]
This PR also collides with https://github.com/publicsuffix/list/pull/1957. The whole block for `Wix.com, Inc.` is in the wrong spot. It should be located below `WISP` but is actually located below `WP Engine`.

**Comment by yahesh** [comment]
Sorting looks good to me now.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15564,6 +15564,14 @@ wmcloud.org
 panel.gg
 daemon.panel.gg
 
+// Wix.com, Inc. : https://www.wix.com
+// Submitted by Shahar Talmi <shahar@wix.com>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:15568]: Please use a non-personal e-mail.

**Comment by simon-friedberger** [comment]
* [ ] Expiration (Note: Must STAY >2y at all times)
  * [x] wixstudio.com expires 2027-12-17
  * [x] wixsite.com expires 2026-05-08
  * [ ] editorx.io expires 2025-03-01
  * [x] wixstudio.io expires 2026-05-15
  * [ ] wi.run expires 2025-01-18
* [ ] DNS _psl entries (Note: Must STAY in place)
  * [x] wixstudio.com
  * [ ] others are missing
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description

**Comment by alonkochba** [comment]
Thanks, I am only adding wixstudio.com here, the rest is refactoring, but I will reach out to make sure the others are extended as well and PSL is readded.

**Comment by simon-friedberger** [comment]
@alonkochba Any updates?

**Comment by alonkochba** [comment]
Hi, all comments done

1. domains expiration updated
2. _psl DNS entries added for all domains pointing to this PR
3. changed mail to publicsuffixlist@wix.com

File: public_suffix_list.dat (hunk 1 above)
Comment (alonkochba) [reply public_suffix_list.dat]: done, added both our names and changed to publicsuffixlist@wix.com

**Comment by alonkochba** [comment]
PR check seems to fail on someone else's entry

> Checking TXT entry for _psl.emrnotebooks-prod.ap-south-2.amazonaws.com.

**Comment by simon-friedberger** [comment]
> PR check seems to fail on someone else's entry, any idea why?
> 
> > Checking TXT entry for _psl.emrnotebooks-prod.ap-south-2.amazonaws.com.
> > DNS answer: "#1919" -> PR 1919

I'm working on it, don't worry about it. It's not your fault. :)

**Pull request merged**